src/apis/AppState/index.js

```diff
@@ -2,7 +2,7 @@ import ExecutionEnvironment from 'fbjs/lib/ExecutionEnvironment';
 import findIndex from 'array-find-index';
 import invariant from 'fbjs/lib/invariant';
 
-const EVENT_TYPES = [ 'change' ];
+const EVENT_TYPES = ['change'];
```

Comment:
We used to add spaces around [] in the bracket spacing option but a lot of people complained so we removed it. May be worth revisiting. cc @jlongster for a data point.